packages/react/src/River/River.module.css

```diff
@@ -111,3 +112,71 @@
     margin-top: var(--base-size-48);
   }
 }
+/* end River */
+
+/* RiverBreakout */
```

Comment:
Related to this comment https://github.com/primer/brand/pull/394/files#r1294572488, can we move this out to a separate component please?